modules/mobile/spec/requests/mobile/v1/health/immunizations_spec.rb

```diff
@@ -2,11 +2,13 @@
 
 require_relative '../../../../support/helpers/rails_helper'
 
-RSpec.describe 'Mobile::V1::Health::Immunizations', :skip_json_api_validation, type: :request do
+RSpec.describe 'Mobile::V1::Health::Immunizations', :openapi_schema_validation, :skip_json_api_validation,
+               type: :request do
   include JsonSchemaMatchers
 
   let!(:user) { sis_user(icn: '9000682') }
   let(:rsa_key) { OpenSSL::PKey::RSA.generate(2048) }
+  let(:default_params) { { page: { size: 100 } } }
```

Comment:
Using this made the specs a little cleaner IMO because most of our tests really just want all records which wasn't always clear.